### ISSUE TITLE
[FORMATTING] when set Indentation style to "Tabular, Right", it doesn't format "between...and" well.

### ISSUE THREAD

**Issue by AndyGeng123** [sql-formatter-org/sql-formatter#341]
SQL:
```
select * from andy.t_test where id between 1 and 5000;
```

When set Indentation style to "Tabular, Right", output from demo website:
```sql
   select *
     from andy.t_test
    where id between 1       and 5000;
```
Many thanks for your great work!

**Comment by inferrinizzard** [comment]
Hi, can you please provide a code snippet so we can see the input and expected outputs

**Comment by AndyGeng123** [comment]
Hi inferrinizzard,
In demo https://sql-formatter-org.github.io/sql-formatter/
Just input the following SQL:
select * from andy.t_test where id between 1 and 5000;
Then set Indentation style to "Tabular, Right".
You can see before "and", there are about 7 spaces.
Thanks.

**Comment by nene** [comment]
Edited the original to use code blocks.

**Comment by nene** [comment]
Fix now released in 9.0.0-beta4.
